packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -61,13 +61,13 @@ export default mixins(
     dayFormat: Function as PropType<DatePickerAllowedDatesFunction | undefined>,
     disabled: Boolean,
     events: {
-      type: [Array, Function, Object] as PropType<DatePickerEvents>,
+      type: [Array, Function, Object],
       default: () => null,
-    },
+    } as PropValidator<DatePickerEvents>,
```

Comment:
```suggestion
    } as PropValidator<DatePickerEvents | null>,
```

Reply:
We should really be doing this pretty much everywhere, most props default to `undefined`